public/index.php

```diff
@@ -3,3 +3,18 @@
 
 throw new \RuntimeException('Insert your API key'); // REMOVE THIS
 $ably = new \Ably\AblyRest("{{ApiKey}}");
+
+echo "<pre>\n";
+
+echo "Retrieving channel...\n";
```

Comment:
Nothing is actually being retrieved as such, especially in the stateless REST library i.e. nothing actually happens at this point other than instancing a channel.  I would either remove this comment or change it to be more accurate such as "Instancing a channel object"